framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1300,8 +1300,42 @@ private boolean methodIsStaticallyExecutable(Element method) {
             return getDeclAnnotation(method, StaticallyExecutable.class) != null;
         }
 
+        /**
+         * @return The Range of the Math.min or Math.max method, or null if the argument is none of
+         *     these methods or their arguments are not annotated with @IntRange.
+         */
+        private Range getRangeForMathMinMax(MethodInvocationTree tree) {
+            if (getMethodIdentifier().isMathMin(tree, processingEnv)) {
+                AnnotatedTypeMirror arg1 = getAnnotatedType(tree.getArguments().get(0));
+                AnnotatedTypeMirror arg2 = getAnnotatedType(tree.getArguments().get(1));
+                if (arg1.hasAnnotation(IntRange.class) && arg2.hasAnnotation(IntRange.class)) {
```

Comment:
This logic belongs in the `org.checkerframework.common.value.util.Range` class.  You should add a `Range min(Range other)` and a `Range max(Range other)` methods.